cli/PrintToScreen.cpp

```diff
@@ -147,4 +148,25 @@ void PrintToScreen::printTuple(const TupleStorageSubBlock &tuple_store,
   fputc('\n', out);
 }
 
+std::size_t PrintToScreen::GetNumTuplesInRelation(
+    const CatalogRelation &relation, StorageManager *storage_manager) {
+  const std::vector<block_id> &blocks = relation.getBlocksSnapshot();
+  std::size_t total_num_tuples = 0;
+  for (block_id block : blocks) {
+    total_num_tuples +=
+        storage_manager->getBlock(block, relation)->getNumTuples();
+  }
+  return total_num_tuples;
```

Comment:
@hbdeshmukh This feature is great, but I suggest to merge this method as a part of `PrintToScreen::PrintRelation` around https://github.com/pivotalsoftware/quickstep/blob/master/cli/PrintToScreen.cpp#L94.

And thus the implementation of `PrintToScreen::PrintOutputSize` could just append to `PrintToScreen::PrintRelation` as the last line.